### DISCUSSION TITLE
hands not appearing, able to use joystick for navigating the menu

### DISCUSSION THREAD

**Discussion by IronVR23** [Okabintaro/SubmersedVR#66] [Q&A] [answered]
I'm on quest 2 air link, and i looked at the readme. I followed the tutorial from imcharliedelta (link https://www.youtube.com/watch?v=935AUfqPI3o) and he did it a bit different from the readme, but it still worked. do i also need to follow the development setup instructions? charliedelta did't do any of them (i think) and his still worked, but hes also on vive.

**Comment by Okabintaro** [comment]
If you want to play on quest you have to follow the readme yes. Especially the last point: https://github.com/Okabintaro/SubmersedVR#installationsetup

**Reply by IronVR23** [reply]
think i might have found the problem, do i also have to deploy the mod in vortex?
i did do the -vrmode openvr thing, and it's still not working.

**Reply by Okabintaro** [reply]
Maybe, I never used vortex myself so I am not sure.
You should see the BepInEx Console poping up and there should a log line with Submersed VR in there.

**Reply by IronVR23** [reply]
that did happen when i modded stardew valley, but what did you use to mod Submersed?
Should I delete the files and unmod, then drop the whole zip file inside of the subnauitca folder from the readme?

**Reply by IronVR23** [reply]
also, the development setup is kinda confusing

**Reply by Okabintaro** [reply]
If you just wanna play the game you don't have to follow the development setup.

Yes, you just have to drop the contents of the zip into the Subnautica folder after installing BepInEx.
See other discussions like this: https://github.com/Okabintaro/SubmersedVR/discussions/50

**Reply by IronVR23** [reply, marked as answer]
I looked at a different tutorial, and I got it working! deltacharlie had me take all of the contents and put them in different places, but all i had to do was drop the stuff in my subnauitca folder. thanks for the help!
(anyone looking at this afterwards, follow the tutorial by Virtual panic (link https://www.youtube.com/watch?v=4OSVtCSQpOE&t=250s))

**Reply by RedstoneManiac3Xbox** [reply]
hey, im prolly just stupid, but i have everything in the right place, and i launch steam vr from within oculus link and then start subnautica, but nothing is working

**Comment by IronVR23** [comment]
did you go into your lauch settings and put in the steam run vr thingy? if
so than i have no idea, just look at the tut i referenced

**Reply by RedstoneManiac3Xbox** [reply]
Yea I launched it via steam vr, but I was launching steamvr itself from the oculus runtime.  What I had to do was swap from oculus link to Virtual desktop and launch steam vr that way to get it to work

(I'm mad but I understand sorta how limitations work. I just wish I figured out it was oculus link causing the issue before I reinstalled 3 times. The other thing is if I hadn't already bour VD for other reasons, someone else in my position who hadn't bought it already would be out $20)

**Reply by Thedemon126** [reply]
this is so confusing... so that means no subnautica vr for me?

**Reply by RedstoneManiac3Xbox** [reply]
If you purchase virtual desktop $20 on oculus mp, and have good wifi, you can play it. But AFAIK if you try to run it over oculus link, regardless of how you start the game. You can't have this hands mod
